src/script/lua_api/l_util.cpp

```diff
@@ -394,7 +394,8 @@ int ModApiUtil::l_decode_base64(lua_State *L)
 	const std::string data = std::string(d, size);
 
 	if (!base64_is_valid(data))
-		return 0;
+		lua_pushnil(L);
+		return 1;
```

Comment:
Your IDE should highlight that this indent level is misleading. This code block needs surrounding curly brackets.

Reply:
It doesn't for some reason, maybe because of this error:

![image](https://github.com/minetest/minetest/assets/48543043/ff4b8846-754a-4e99-bac7-822982e655d2)

Reply:
This option is supported since clang-format 13. The newest documentation about this option is very misleading because it previously only accepted booleans.